### DISCUSSION TITLE
我不管，我就要离线版

### DISCUSSION THREAD

**Discussion by Zoltan-code** [the1812/Bilibili-Evolved#3476] [功能建议 / Ideas]
对我来说，太难了，求求了，搞个离线版吧
或者让墙内的人也能访问到

**Comment by Zoltan-code** [comment]
不给加？那我就在这里贴上1.0最后一个版本，里面有离线版，真的是救了我的狗命了
https://github.com/the1812/Bilibili-Evolved/tree/v1.12.22

**Reply by Geeyun-JY3** [reply]
最后一个预览离线版一直留着的……：https://github.com/the1812/Bilibili-Evolved/blob/master/bilibili-evolved.offline.user.js
